src/test/resources/features/99f_state_benefits_qa_smoke.feature

```diff
@@ -30,21 +30,21 @@ Feature: State Benefits Journeys - Income Tax Submission
     And the user selects the claim month field and enters a value of 01
     And the user selects the claim year field and enters a value of 2022
     And the user clicks the continue button
-    Then the user should be redirected to the "Did this claim end between 1 January 2022 and 5 April 2022?" page
+    Then the user should be redirected to the "Did this claim end between 6 April 2022 and 5 April 2023?" page
```

Comment:
maybe we could update the year of the start date here instead? So we only update the years in the rest of places?